libs/output-mapping/src/Writer/Helper/SliceHelper.php

```diff
@@ -117,13 +119,23 @@ public function sliceFile(MappingSource $source): MappingSource
             $sourceFile->getFile(),
             $outputDir,
             $source->getManifestFile(),
+            $this->inputSizeThreshold,
         );
-        $process->mustRun(function ($type, $buffer) {
+
+        $result = $process->run(function ($type, $buffer) {
             if ($type === Process::OUT) {
                 $this->logger->info(trim($buffer));
             }
         });
 
+        if ($result === SliceCommandBuilder::SLICER_SKIPPED_EXIT_CODE) {
+            throw new SliceSkippedException('No need to slice, the source data is not large enough.');
```

Comment:
přidat filename ?

Reply:
V ostatních případech to taky není. Není potřbeba. To dělá sliceSources:

https://github.com/keboola/platform-libraries/blob/4e0095528730bf5057043cdcd7bcc7b7ea630990/libs/output-mapping/src/Writer/Helper/SliceHelper.php#L87-L99

Nestuduj jen změněný kód :) to je cesta do pekel :) obzvláště v OM

Reply:
oukej :)